fe/src/main/java/org/apache/doris/master/MasterImpl.java

```diff
@@ -559,9 +559,17 @@ private void finishPublishVersion(AgentTask task, TFinishTaskRequest request) {
         if (request.isSetError_tablet_ids()) {
             errorTabletIds = request.getError_tablet_ids();
         }
+
+        if (request.isSetReport_version()) {
+            // report version is required. here we check if set, for compatibility.
+            long reportVersion = request.getReport_version();
+            Catalog.getCurrentSystemInfo().updateBackendReportVersion(task.getBackendId(), reportVersion, task.getDbId());
+        }
```

Comment:
report_version maybe rollback here?

Reply:
It won’t, updateBackendReportVersion will only increase the version